magento/pwa-studio
# [Theming] Refactor site header

## Description

Refactors the site header component to use Tailwind styles.

Not included in the scope of this change are some of the sub-modules inside the Header directory and other UI components used inside the Header component.

## Related Issue







Closes PWA-1885

## Acceptance





@schensley 
@tkacheva 

### Verification Stakeholders


@jimbo 
Core developers

### Specification




## Verification Steps

1. Run the develop server
2. Inspect the Header component elements
3. Verify tailwind classes are listed in the `class` property of the elements






#### Test scenario(s) for direct fix/feature

1. Verify Header is rendered across all viewports




#### Test scenario(s) for any existing impacted features/areas

1. Verify Header buttons still work as expected
1. Verify Mega Menu still works
2. Verify Search stills works





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?

Yes, device testing across all relevant viewports is needed since the Header UI component styles have been refactored.





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
